src/basicElements/serializers/number/integers.ts

```diff
@@ -67,3 +78,44 @@ export function numberToUnsigned(
   mustBeValidUnsigned(sizeInBits, int)
   return int
 }
+
+function setBigUint128(
+  view: DataView,
+  byteOffset: number,
+  value: bigint
+): void {
+  view.setBigUint64(byteOffset, value & U64.MAX, true)
+  view.setBigUint64(
+    byteOffset + U8.SIZE_BIT,
+    value >> BigInt(U64.SIZE_BIT),
+    true
+  )
+}
+
+function getBigUint128(view: DataView, byteOffset: number): bigint {
+  const low = view.getBigUint64(byteOffset, true)
+  byteOffset += U8.SIZE_BIT
+  const high = view.getBigUint64(byteOffset, true)
+  return (high << BigInt(U64.SIZE_BIT)) | low
+}
+
+function setBigUint256(
+  view: DataView,
+  byteOffset: number,
+  value: bigint
+): void {
+  for (let i = ZERO; i < U256_NUM_64_BIT_CHUNKS; i++) {
```

Comment:
this constant is unclear. You should either:
- directly use `U256.SIZE_BYTE / U64.SIZE_BYTE`
- define a constant named U256_SIZE_IN_U64 intialized to this value